ckanext/data_qld/tests/test_reporting.py

```diff
@@ -104,8 +104,7 @@ def test_as_regular_user(self, app, user):
         app.get('/', extra_environ={"REMOTE_USER": str(user["name"])})
```

Comment:
can you update test method name, 
test_as_regular_user_is_blocked_unauthorized

Reply:
Renamed.